### PR TITLE
make pinbot require a label on pins

### PR DESCRIPTION
**Pull request by whyrusleeping**
This will make it a little easier to keep track of whats been pinned

### CODE REVIEW

**Comment by whyrusleeping** [comment]
cc @lgierth

File: main.go [hunk 1]
```diff
@@ -63,7 +63,22 @@ func tryUnpin(path string, sh *shell.Shell) error {
...
-func Pin(b *hb.Bot, actor, path string) {
+var pinfile = "pins.log"
+
+func writePin(pin, label string) error {
+	fi, err := os.OpenFile(pinfile, os.O_APPEND|os.O_EXCL|os.O_WRONLY, 0660)
+	if err != nil {
+		return err
+	}
+
+	_, err = fmt.Fprintf(fi, "%s\t%s\n", pin, label)
```
Comment (ghost) [inline main.go:74]: can irc messages contain whitespace? if they do, we should escape it, so that we don't need to take care of it when parsing the log

**Comment by ghost** [comment]
Tested and LGTM :+1: 
- [ ] whitespace question

File: main.go (hunk 1 above)
Comment (whyrusleeping) [inline main.go:74]: all whitespace after the beginning command is preserved, this should be good

**Pull request merged**